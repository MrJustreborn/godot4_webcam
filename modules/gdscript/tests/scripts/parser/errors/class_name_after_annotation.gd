# Error here. `class_name` should be used *before* annotations, not after.
@icon("res://path/to/optional/icon.svg")
class_name HelloWorld

func test():
	pass
