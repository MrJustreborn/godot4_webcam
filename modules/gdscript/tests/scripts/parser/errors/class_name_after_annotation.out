GDTEST_PARSER_ERROR
"class_name" should be used before annotations.
